doc/nrf/protocols/wifi/sniffer_rx_operation.rst

```diff
@@ -0,0 +1,136 @@
+﻿.. _ug_nrf70_developing_raw_ieee_80211_packet_reception:
+
+Raw IEEE 802.11 packet reception using Monitor mode
+###################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF70 Series device supports Monitor mode, that is, listening for and receiving all IEEE 802.11 traffic on a given channel, not restricted to a certain SSID. With Monitor mode raw IEEE 802.11 packets can be received directly at the user application layer.
```

Comment:
Please ensure "Monitor mode" is used throughout.

Also, Sachin's suggestion can be further condensed to the following:

```suggestion
The nRF70 Series devices support Monitor mode.
In this mode, the device listens on a configured channel and receive all IEEE 802.11 traffic, including traffic from all BSSIDs operating on that channel.
The device then prepends the captured traffic with metadata before passing it to the application layer using raw sockets.
```

Reply:
I think most of this is repeated below (e.g. the thing about raw sockets, packet meta-data, etc.). Therefore, I envisioned a simpler and more high-level description here.

Reply:
Yes, earlier it was a more elaborate description at the top. Hence, as per discussion, it was changed.